### PR TITLE
fix: bignumber comparison label layout

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: https://github.com/lightdash/lightdash/issues/9471

### Description:

https://github.com/lightdash/lightdash/assets/962095/6aba6d56-1821-4260-9605-ba4fb6b3ed55

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/SimpleStatistic/index.tsx [hunk 1]
```diff
@@ -159,79 +160,116 @@ const SimpleStatistic: FC<SimpleStatisticsProps> = ({
...
+                        fz={valueFontSize}
+                        c="dark.4"
+                        align="center"
+                        fw={500}
+                        style={{
+                            transition: 'font-size 0.1s ease-in-out',
+                        }}
+                    >
+                        {bigNumber}
+                    </Text>
```
Comment (IrakliJani) [inline packages/frontend/src/components/SimpleStatistic/index.tsx:183]: these `<Text` components repeat a lot. maybe it's better to create wrapper components?

Comment (IrakliJani) [reply packages/frontend/src/components/SimpleStatistic/index.tsx]: done.

**Review by TuringLovesDeathMetal — APPROVED** [review]
_**so**_ much better, thank you!

**Review by magnew — COMMENTED** [review]
Definitely works in dashboards better! The layout in the explore view looks different though, do we want that? Text is smaller and positioned at the top. And the gap to the compare seems odd. 

Explore page in MAIN:
Screenshot 2024-03-21 at 17 56 06

Same chart with this patch:
Screenshot 2024-03-21 at 17 56 15

**Review by magnew — APPROVED** [review]
Update looks good

**Comment by IrakliJani** [comment]
profiles e2e is failing which was recently fixed and is on main. merging.

**Pull request merged**